util/metrics/src/service_metrics.rs

```diff
@@ -187,6 +194,7 @@ impl Collector for ServiceMetrics {
 
 /// This method reads the full URI from gRpcContext (looks like:
 /// `/{package}.{service_name}/{method}`
+/// ('/' equates to ascii code 47)
 /// and converts it into a dot-delimited string, dropping the 1st `/`
```

Comment:
nit: it feels like the notation about the ascii code interrupts the thought.

```suggestion
/// and converts it into a dot-delimited string, dropping the 1st `/`
/// ('/' equates to ascii code 47)
```

Reply:
Was this suggestion meant to be committed, or ignored?